app/src/main/java/com/example/tasky/feature_agenda/data/remote/dto/AgendaResponse.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.tasky.feature_agenda.data.remote.dto
+
+import com.example.tasky.feature_agenda.domain.model.Event
+import com.example.tasky.feature_agenda.domain.model.Reminder
+import com.example.tasky.feature_agenda.domain.model.Task
```

Comment:
If you work with presentation/domain/data layers, you'd need to introduce DTOs specific for the data layer (e.g. `EventDto`) which just represent the model from the API. Then you can have a mapper function that maps these to the domain model.
Right now what you have is not ideal since the names of the domain models need to match the names of the JSON exactly. The domain models are rather objects that can be conveniently used across the project and everything is already mapped/parsed to helpful fields (for example `LocalDateTime` instead of a `Long` timestamp)